staging/src/k8s.io/client-go/testing/fixture_test.go

```diff
@@ -278,12 +280,17 @@ func TestPatchWithMissingObject(t *testing.T) {
 func TestGetWithExactMatch(t *testing.T) {
 	scheme := runtime.NewScheme()
 	codecs := serializer.NewCodecFactory(scheme)
+	namer := namer.NewAllLowercasePluralNamer(nil)
```

Comment:
using a namer without configurable exceptions just bakes in the special cases at another layer... same comment applies everywhere we're using `NewAllLowercasePluralNamer(nil)`